src/transport/SessionHandle.cpp

```diff
@@ -23,6 +23,15 @@ namespace chip {
 
 using namespace Transport;
 
+using SubjectDescriptor = Access::SubjectDescriptor;
```

Comment:
So this defines a `chip::SubjectDecriptor`, but only if you include this header.  Is that intended?

Reply:
This is in the .cpp
Will it have external linkage?
If so I'd rather just scope it where it's used.

Reply:
Oh, so it is.  I don't know why I thought this was in a header.  :(  This part is fine.